UploadServlet.java

```diff
@@ -55,6 +60,43 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response) t
 
 	protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
 		
+		String author = request.getParameter("author");
+		String subject = request.getParameter("subject");
+		String title = request.getParameter("title");
+		String highlights = request.getParameter("highlights");
+		String abstracts = request.getParameter("abstracts");
+		
+		
+		DB db = new DB();
+		boolean checkstatus = false;

```

Comment:
@hema-001 

I think `duplicate_article_title` would be a better variable name.  Why?  More specific on the purpose of declaring
this variable.

**Hui**
Hui

Reply:
I totally agree. but, I was focusing on fixing the format problem I didn't notice this line.

IBRAHIM